PureSpinningLocalPlanner:
    yaw_goal_tolerance: 0.02
    intermediate_goals_yaw_tolerance: 0.04
    k_betta: 20.0
    max_angular_z_speed: 2.0 # rad/s